src/backend/distributed/sql/udfs/citus_prepare_pg_upgrade/latest.sql

```diff
@@ -17,6 +17,14 @@ BEGIN
 
     DROP AGGREGATE IF EXISTS array_cat_agg(anyarray);
     DROP AGGREGATE IF EXISTS array_cat_agg(anycompatiblearray);
+
+    -- We should drop any_value because PG16 has its own any_value function
```

Comment:
maybe expand the comment saying that once we do not support PG16, we can drop this code, and the corresponding code in post upgrade script

Reply:
Hmmm, not that you say this, we should probably drop the corresponding codes for upgrading to PG14 (the example I referred to https://github.com/citusdata/citus/commit/8ef94dc1f503d14451f65d1574973bfc3840b7c1)
since we dropped PG13 support. Will open an issue to keep this as a note.

Reply:
> we should probably drop the corresponding codes for upgrading to PG14

Or wait - maybe we should wait until we drop pg14 support as well?

Reply:
> Or wait - maybe we should wait until we drop pg14 support as well?

I think so